test/controllers/shared/basic_rest_response_test.rb

```diff
@@ -68,6 +68,15 @@ def basic_pagination_per_page_test
       end
     end
 
+    def all_per_page_test
+      test 'should render all records' do
+        get :index, params: { per_page: 'all' }
+        assert_response :success
+        res = ActiveSupport::JSON.decode(@response.body)
+        assert_equal @controller.controller_name.classify.constantize.count, res['results'].size
```

Comment:
Are we sure the amount of resources here are more than the default per_page? Otherwise you might be testing something and an invalid `per_page` could simply be ignored.

Reply:
I've pushed a setup for this test, that creates the records if those are not there. To include more records than `per_page` default. Is it ok like this?